package.json

```diff
@@ -58,7 +58,7 @@
     "url": "https://github.com/balena-io/balena-preload/issues"
   },
   "scripts": {
-    "lint": "balena-lint -e js --fix lib",
+    "lint": "balena-lint --fix lib",
```

Comment:
This change allows balena-lint to actually run on typescript files. This commit runs the linter, hence many changes.